### ISSUE TITLE
propagate random seed to the model

### ISSUE THREAD

**Issue by HuStmpHrrr** [metab0t/PyOptInterface#34]
In my application, I observe that setting random seed in python is not sufficient to control the behavior of the app. I suspect that MILP solving might produce a different model sometimes. Is there a way to set the random seed, or you are doing it already?

**Comment by metab0t** [comment]
I do not think that setting random seed in Python will affect the behavior of solver.

Maybe you should set the parameters of solver to control the MIP optimization process.
